### PR TITLE
add version and have zopen know about itself to eliminate need for .env

### PR DESCRIPTION
**Pull request by MikeFultonDev**
@IgorTodorovskiIBM and @DevonianTeuchter please review. 
Looking to simplify steps to get started with zopen

I also bumped the version to 1.0.1 so there isn't any confusion about what is 'better' since we have an old 1.0.0

### CODE REVIEW

File: bin/lib/zopen-version [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+echo "zopen 1.0.1"
```
Comment (IgorTodorovskiIBM) [inline bin/lib/zopen-version:3]: Wonder if we should have a seperate ZOPEN_VERSION file that can be referenced here and potentially in other places or we expect everything will go through zopen-version?

Comment (IgorTodorovskiIBM) [reply bin/lib/zopen-version:3]: Also, 1.0.1 seems like a big jump or 0.6.3. Should we go with 0.7.0?

Comment (MikeFultonDev) [reply bin/lib/zopen-version:3]: i was thinking all zopen tools can just call zopen-version to print out the version so we don't have to worry about keeping things in sync

1.0.1 is not much of a jump from 1.0.0 (which we did before the 0.6.x)

happy to make it 0.6.4 if we delete the 1.0.0 pax file

Comment (MikeFultonDev) [reply bin/lib/zopen-version]: added code back in for .env with comment to indicate it isn't required for consumers
changed bump to 0.7.0 instead of 1.0.1

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**